jest.config.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     // This exclusion avoids running coverage on Barrel files, https://twitter.com/housecor/status/981558704708472832
     '!app/javascript/**/index.js',
     '!app/javascript/packs/**/*.js', // avoids running coverage on webpacker pack files
+    'app/javascript/packs/dashboards/*.js', // try opting-in for jest coverage?
```

Comment:
:monocle_face: :thinking: Is this override something we want permanently? Especially in light of the Slack discussion this morning about flexing coverage % when it's less feasible to get high numbers...

Reply:
It's a fair question. Primarily, there's some lag here — the PR has already been open a couple of days, and that discussion is still pretty new. I'm not sure, yet, where that discussion will ultimately land.

Avoiding coverage on pack files seems _mostly_ to be about us wanting to avoid paying down all that tech debt at once. The side-effect seems to be artificially _low_ coverage, when we _do_ actually pay down that debt.

It was tempting to organize this into `packs/legacy` (exempted from coverage) and `packs/modern` (with coverage), but I thought that might deserve its own PR.